src/network/netdev/netdev.c

```diff
@@ -995,15 +995,12 @@ int netdev_load_one(Manager *manager, const char *filename) {
         return 0;
```

Comment:
something went wrong with the commit msg? the first and 2nd line seem duplicative?

Reply:
I do not think so? Explained two functions. I will rephrase later.